server/src/BitBakeDocScanner.ts

```diff
@@ -111,7 +112,10 @@ export class BitBakeDocScanner {
   }
 
   public setDocPathAndParse (extensionPath: string): void {
-    this._docPath = path.join(extensionPath, 'resources/docs')
+    // https://github.com/yoctoproject/poky/tree/master/bitbake/doc/bitbake-user-manual
```

Comment:
For VSCode, we bundle the documentation inside our VSCE package to get a stable experience. I advise to do the same for nvim. You shouldn't need to edit these paths as a consequence.

Reply:
OK, It will be on another PR.